lnst/RecipeCommon/Perf/Measurements/RDMABandwidthMeasurement.py

```diff
@@ -0,0 +1,135 @@
+from dataclasses import dataclass
+from typing import Any, Optional
+import time
+
+from lnst.Common.IpAddress import BaseIpAddress
+from lnst.Controller.Host import Host
+from lnst.Controller.Job import Job
+from lnst.Controller.Recipe import BaseRecipe
+from lnst.Controller.RecipeResults import ResultType
+from lnst.Tests.RDMABandwidth import RDMABandwidthServer, RDMABandwidthClient
+from lnst.RecipeCommon.Perf.Measurements.BaseMeasurement import BaseMeasurement
+from lnst.RecipeCommon.Perf.Measurements.Results import (
+    RDMABandwidthMeasurementResults,
+    AggregatedRDMABandwidthMeasurementResults
+)
+
+
+@dataclass
+class RDMAFlow:
+    generator: Host
+    receiver: Host
+    generator_nic_name: str
+    receiver_nic_name: str
+    generator_bind: BaseIpAddress
+    receiver_bind: BaseIpAddress
+
+
+@dataclass
+class RDMAFlowTest:
+    flow: RDMAFlow
+    server_job: Job
+    client_job: Job
+
+
+class RDMABandwidthMeasurement(BaseMeasurement):
+    def __init__(
+        self,
+        flows: list[RDMAFlow],
+        recipe_conf: Any = None,
+    ):
+        super().__init__(recipe_conf)
+
+        self._flows = flows
+        self._running_flow_tests: list[RDMAFlowTest] = []
+        self._finished_flow_tests: list[RDMAFlowTest] = []
+
+    @property
+    def flows(self) -> list[RDMAFlow]:
+        return self._flows
+
+    def start(self) -> None:
+        self._running_flow_tests = self._prepare_flow_tests()
+
+        for flow_test in self._running_flow_tests:
+            flow_test.server_job.start(bg=True)
+
+        time.sleep(2)
+
+        for flow_test in self._running_flow_tests:
+            flow_test.client_job.start(bg=True)
+
+    def finish(self) -> None:
+        for flow_test in self._running_flow_tests:
+            flow_test.client_job.wait()
+            flow_test.server_job.wait()
+
+        self._finished_flow_tests = self._running_flow_tests
+        self._running_flow_tests = []
```

Comment:
I don't undersrand what the `self._running_flow_tests` and `self._finished_flow_tests` are for.

Even the initialization of `self._running_flow_tests` does not quite match the name variable. The tests are not running yet.

With the current code it would be sufficient to have just `self._flow_tests`. Or rename `running_tests` to `self._flow_tests`  and have `self._finished_tests`. But still, there's no use of the variable other than tracking ALL of the flow_tests.

Reply:
It was because the list doesn't get cleared on more perf iterations. I clear the list now and moved to using a single list.